subjects/make-your-game/make-your-game-score-handling/README.md

```diff
@@ -0,0 +1,86 @@
+## scoreboard
+
+### Objectives
+
+You must follow the same [principles](https://public.01-edu.org/subjects/make-your-game/README.md) as the first subject.
+
+For this project you must take into account:
+
+- The usage of **scoreboards**
+- Creation of a **go API service** to save the data from the game into a JSON file
```

Comment:
we should ask for `JSON` format but not a `JSON` file

Reply:
ok, replaced the JSON files to JSON format